plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -184,7 +193,7 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
         const value: number = !richTooltip ? params.value : params[0].value[0];
         const prophetValue = !richTooltip ? [params] : params;
 
-        const rows: Array<string> = [`${xAxisFormatter(value)}`];
+        const rows: Array<string> = [`${tooltipFormatter(value)}`];
```

Comment:
I tested and found that the `String` constructor did not work.

![image](https://user-images.githubusercontent.com/2016594/114038616-9a962b00-98b4-11eb-8083-74f8062e18a0.png)